src/sparseml/transformers/utils/helpers.py

```diff
@@ -75,6 +75,22 @@ class TaskNames(Enum):
 }
 
 
+def remove_past_key_value_support_from_config(config: AutoConfig) -> AutoConfig:
+    """
+    Modify config of the causal language model so that it turns off the
+    past key value support. This means that the model initialized from
+    this config will not take past key values as input and will not output
+    past key values.
+    """
+    # not take past_key_values as input
+    config.is_decoder = True
+    # whether to use past key values an input
+    config.use_past = False
+    # whether to output past key values
+    config.use_cache = False
```

Comment:
is there potential to fail if these members don't exist?

Reply:
Have you seen an instance of a config of a CausalModel (text-generation) that did not have those attributes? They are pretty standard no?

Reply:
yeah i was just thinking from a possible null access perspective, but that doesn't matter for python members